### ISSUE TITLE
The normal-BAF filter fits a beta-binomial and discards most of it, in both references

### ISSUE THREAD

**Issue by michaelJwilson** [michaelJwilson/port#183]
**`normal_baf_bin_filter` fits a one-state beta-binomial to the pooled normal-spot counts, then overwrites the fitted success probability with 0.5 and floors the concentration at 30. CalicoST does exactly the same. Whether that is a deliberate prior or a leftover is a paper question, and it is the one thing #38 asked that closing #38 would lose.**

```python
# cnaster normal_spot.py:970-976        CalicoST utils_IO.py:977-980
tmpres = model.fit(**settings)          tmpres = model.fit(...)
# TODO warn if patched.
tmpres.params[0] = 0.5                  tmpres.params[0] = 0.5
tmpres.params[-1] = max(tmpres.params[-1], min_betabinom_tau)   # identical
```

A fit is performed, its first parameter is discarded unconditionally, and its last is floored — under `# TODO warn if patched`, which is not done. The interval is then computed from parameters that are **half fitted and half asserted**.

## What is and is not settled

[#164](https://github.com/michaelJwilson/port/pull/164) delivers the pin #38 asked for: at the shipped interval the filter removes exactly the bins the planted normal clone carries an event in, eight of forty, no bin either way. So the filter **works** under the patched parameters on a fixture where the normal clone really is balanced.

What that does not establish is whether the patch is right when it is not. Forcing `p = 0.5` says "the normal population is balanced by definition"; the whole point of the filter is to catch a normal clone that is **erroneously identified or mixed with non-normal spots**, which is the function's own docstring. Under that failure mode the fitted `p` is the evidence, and it is the value being thrown away.

The `tau >= 30` floor is the same shape: it widens the interval when the fit says the pooled counts are more concentrated than that, which makes the filter more conservative — deliberately, or because the fit is unstable at one state. Nothing says which.

## CalicoST: inherited, with a wider interval

Identical patching. The shipped intervals differ: CalicoST defaults to `[0.05, 0.95]` (`utils_IO.py:970`), `cnaster` reads `quality.normal_allele_specific_confidence`, which `zenodo_sim_config.yaml` sets to `(0.01, 0.99)`. So `cnaster` removes strictly fewer bins at the same data — a shipped-value change, not a code one.

## Done when

`emission.tex` is read for what the normal-spot test is supposed to condition on, and the patch is recorded as **a difference in the problem or a defect** per **The three references**. No label: this is a question, not a defect, until the paper says otherwise.

Raised by #38, which [#164](https://github.com/michaelJwilson/port/pull/164) closes on the coverage grounds — this is the part that outlives it. Refs #24, #38, #89, #164, #174.
